src/javascript/webgl-rendering-context.js

```diff
@@ -2386,6 +2391,28 @@ class WebGLRenderingContextHelper extends NativeWebGLRenderingContext {
       data)
   }
 
+  texSubImage3D (target, level, xoffset, yoffset, zoffset, width, height, depth, format, type, pixels) {
+    if (pixels === null || pixels === undefined) {
+      return
+    }
+
+    if (typeof pixels !== 'object') {
+      throw new TypeError('texSubImage3D(GLenum, GLint, GLint, GLint, GLint, GLint, GLint, GLint, GLenum, GLenum, Uint8Array)')
+    }
+
+    if (
+      typeof pixels === 'object' &&
```

Comment:
Likely don't need this line, as it's checked for in the previous `if` statement

Reply:
Done.